contents/data-display/charts/weekly-temperature-chart/metadata.json

```
@@ -9,7 +9,7 @@
   },
   "options": {
     "container": {
-      "p": "md"
+      "minH": "3xl"
     }
```

How about making utils.ts appear before locations.ts?

```suggestion
    },
    "files": {
      "order": ["index.tsx", "utils.ts", "location.ts"]
    }
```

---

Done.